roles/kubernetes/master/defaults/main.yml

```diff
@@ -29,3 +29,4 @@ kube_apiserver_memory_limit: 2000M
 kube_apiserver_cpu_limit: 800m
 kube_apiserver_memory_requests: 256M
 kube_apiserver_cpu_requests: 300m
+kube_apiserver_storage_backend: etcd2
```

Comment:
@ant31 Kubernetes uses etcd3 in etcd2 compatibility mode in 1.5. You have to explicitly enable etcd3 format to use it